docker/Dockerfile

```
@@ -489,9 +489,18 @@ CMD ["/bin/bash"]
 FROM base AS universe
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 ARG ROS_DISTRO
-ARG LIB_DIR
+ARG TARGETPLATFORM
```

```suggestion
ARG TARGET_PLATFORM
```

to keep it consistent, maybe we could rename this

---

TARGETPLATFORM is default argument which is set by docker buildx

---

ok :sweat_smile: